.github/workflows/sourcegenerator-benchmark.yml

```diff
@@ -0,0 +1,50 @@
+name: SourceGenerator Benchmark
+on:
+  push:
+    branches:
+      - main
```

Comment:
It will take a while to un eahc time but I've changed it to:
```yml
workflow_call:
pull_request:
  branches:
    - "**"
push:
  branches:
    - main
```

Reply:
See also https://github.com/benchmark-action/github-action-benchmark#caveats